### PR TITLE
CIA Outfit!

### PR DESCRIPTION
**Pull request by ShiggyPiggy**
see aye eyy
A CIA outfit, used with Skowron sprites, "coded" and added into the IAA's locker!
More fun and feeling in charge for all.

### CODE REVIEW

**Comment by DeityLink** [comment]
>10 commits for 15 lines changed.

pls squash

**Comment by ShiggyPiggy** [comment]
@DeityLink, blame Github for not registering my FUCKING CHANGES AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA.

**Comment by Kurfursten** [comment]
10 commits to add a sprite, ShiggyPiggy is living right.

**Comment by Probe1** [comment]
>15 new lines
>10 commits

I need a picture of this.

File: html/changelogs/ShiggyPiggy.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+author: ShiggyPiggy
```
Comment (PJB3005) [inline html/changelogs/ShiggyPiggy.yml]: Why this file?

**Comment by ShiggyPiggy** [comment]
@PJB3005, I dunno, for future coding projects or something.
I've seen other coders do it and figured it might be useful to make my changelogs easier if I decide to code new exciting features at 1,5lines/commit.

**Comment by ShiggyPiggy** [comment]
Closing and rebasing everything tbh

**Pull request closed without merging**